libcudacxx/include/cuda/__memory_resource/resource_ref.h

```diff
@@ -278,8 +278,9 @@ template <_AllocType _Alloc_type, class _Resource>
 _LIBCUDACXX_INLINE_VAR constexpr _Vtable_store<_Alloc_type> __alloc_vtable =
   _Resource_vtable_builder::template _Create<_Resource, _Alloc_type>();
 
+//! @brief Checks whether a passed in type is a specialization of basic_resource_ref
 template <class>
-_LIBCUDACXX_INLINE_VAR constexpr bool _Is_basic_resource_ref = false;
+_LIBCUDACXX_INLINE_VAR constexpr bool __is_basic_resource_ref = false;
```

Comment:
Suggestion: This seems like an internal trait. Do you still want to publicly document it? I would just use a normal comment `//` here or even omit the comment, since the name is descriptive enough IMO.